File: .indio-backups/server-ts-20260507-002053/src/adapters/calendar.ts
import type { CalendarEvent } from "@indio/contracts";

export class CalendarAdapter {
  async getEventsForToday(now = new Date()): Promise<CalendarEvent[]> {
    const isoDate = now.toISOString().slice(0, 10);

    return [
      {
        id: "standup",
        title: "项目对齐",
        startAt: `${isoDate}T09:30:00.000Z`,
        endAt: `${isoDate}T10:00:00.000Z`
      },
      {
        id: "deep-work",
        title: "深度工作块",
        startAt: `${isoDate}T13:30:00.000Z`,
        endAt: `${isoDate}T15:00:00.000Z`
      }
    ];
  }
}

File: agent/adapters/calendar.py
from __future__ import annotations

from models import CalendarEvent


class CalendarAdapter:
    async def get_events_for_today(self) -> list[CalendarEvent]:
        return [
            CalendarEvent(
                id="focus-block",
                title="深度工作时段",
                start_at="09:30",
                end_at="11:30",
            )
        ]
